readme.md

```diff
@@ -30,6 +31,14 @@ pod 'MapboxDirections.swift', :git => 'https://github.com/mapbox/MapboxDirection
 pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :tag => 'v0.0.4'
 ```
 
+#### [Carthage](https://github.com/carthage/carthage)
+
+Add the following line to your Cartfile:
+
+```ruby
+github "mapbox/MapboxNavigation.swift" "swift3"
```

Comment:
MapboxNavigation.swift doesn’t have a swift3 branch. Did you mean MapboxDirections.swift? Make sure to try integrating MapboxNavigation.swift into a standalone application using Carthage, just in case.